src/app/features/auth/pages/registration/registration-page.component.html

```diff
@@ -0,0 +1,14 @@
+<register-form
+  [loading]="loading$ | async"
+  [validationEmailError]="validationEmailError$ | async"
+  (register)="register($event)"
+  (returnToSignIn)="goToSignIn()"
+>
+  <div class="mb-4 flex items-start">
+    <img alt="" src="/assets/icons/Logo.svg" />
+    <div class="block text-gray-700 text-lg font-bold ml-2 mb-2">Freenance</div>
```

Comment:
We are not focused on HTML/CSS, but text should not be wrapped with `div`s. There're tags made specifically for tags: h1-h6, p, span, etc.